region/maridia/inner-pink/West Cactus Alley Room.json

```diff
@@ -102,7 +103,30 @@
           "nodes": [1],
```

Comment:
could be HiJump or Gravity, but then youll probably be able to just do the gravity variant